shardingsphere-governance/shardingsphere-governance-repository/shardingsphere-governance-repository-provider/shardingsphere-governance-repository-etcd/src/main/java/org/apache/shardingsphere/governance/repository/etcd/EtcdRepository.java

```diff
@@ -129,23 +131,37 @@ private Type getEventChangedType(final WatchEvent event) {
                 return Type.IGNORED;
         }
     }
-    
+
     @Override
     public boolean tryLock(final String key, final long time, final TimeUnit unit) {
-        // TODO
-        return false;
+        try {
+            long leaseId = client.getLeaseClient().grant(etcdProperties.getValue(EtcdPropertyKey.TIME_TO_LIVE_SECONDS)).get().getID();
+            client.getLockClient().lock(ByteSequence.from(key, StandardCharsets.UTF_8), leaseId).get(time, unit);
+            return true;
+            // CHECKSTYLE:OFF
+        } catch (final Exception ex) {
+            // CHECKSTYLE:ON
+            log.error("etcdRepository tryLock error, key = {}, time = {}, unit = {}", key, time, unit, ex);
+            return false;
+        }
     }
-    
+
     @Override
     public void releaseLock(final String key) {
-        // TODO
+        try {
+            client.getLockClient().unlock(ByteSequence.from(key, StandardCharsets.UTF_8)).get(etcdProperties.getValue(EtcdPropertyKey.CONNECTION_TIMEOUT_SECONDS), TimeUnit.SECONDS);
+            // CHECKSTYLE:OFF
+        } catch (final Exception ex) {
+            // CHECKSTYLE:ON
+            log.error("etcdRepository releaseLock error, key = {}", key, ex);
```

Comment:
etcdRepository -> EtcdRepository , key = {} -> key: {}

Reply:
ok i have fixed it